src/10_functions.py

```diff
@@ -9,4 +9,19 @@
 # Print out "Even!" if the number is even. Otherwise print "Odd"
 
 # YOUR CODE HERE
+def is_even(num):
+
+	if num % 2 == 0:
+
+		return True
+
+	return False
```

Comment:
Solid! One thing you may see others do is return` n%2 == 0` which will return a Boolean itself. Either way is valid of course.